src/web/app/components/adding-question-panel/adding-question-panel.component.html

```diff
@@ -1,6 +1,6 @@
-<div class="card-body background-color-light-blue text-center">
+<div class="card-body background-color-light-blue text-center responsive-padding">
   <div ngbDropdown class="d-inline-block">
-    <button id="btn-new-question" class="btn btn-primary" ngbDropdownToggle>Add New Question</button>
+    <button id="btn-new-question" class="btn btn-primary p-1 p-sm-2" ngbDropdownToggle>Add New Question</button>
```

Comment:
Let's remove the changes here (`p-1` and `p-sm-2`). I don't see a need to change the padding within the button, and it makes the buttons inconsistent with the rest of the buttons on the page.

Reply:
Unless we apply the padding change within the buttons then at the instructor help page the buttons are overflowing at  373px 
<img width="341" alt="Screenshot 2023-06-26 at 9 21 20 AM" src="https://github.com/TEAMMATES/teammates/assets/60813273/93a93daa-e9f2-4327-bd41-8d3d37543d93">

Reply:
Thanks @singhabhyudita, sorry for missing that out. Shall we shift the relevant padding change into the `.scss` file since Bootstrap doesn't have the `0.75rem`? I'm looking at something like this:

```
@media (max-width: 576px) {
  // prevent overflow for small screen sizes
  #btn-new-question, #btn-copy-question {
    padding-left: .5rem;
    padding-right: .5rem;
  }
}
```

Of course, do let me know if you have a better solution

Reply:
I think `px-2 px-sm-3` is doing the same thing here and it works fine for both the pages 

<img width="339" alt="Screenshot 2023-06-26 at 1 55 37 PM" src="https://github.com/TEAMMATES/teammates/assets/60813273/b92f8e79-1785-4040-bc1d-20a1da67f811">

Reply:
At 0.5rem the buttons overflow at 357px in the instructor help page, so I think 0.75rem will not work for us given that we support a minimum width of 360px.

Reply:
Perhaps my previous comment was unclear - we're trying to solve overflow (for screen widths >= 360px) while ensuring that styles remain consistent throughout the website as much as possible.

The code block I copied above prevents overflow for smaller screen sizes by setting the `padding-left` and `padding-right` to `.5rem` for screen widths <= 576px. Since this styling is only applied for those screen widths, it also maintains consistency by having the button padding revert back to its original padding.

In contrast, `px-2 px-sm-3` doesn't maintain consistency for larger screen widths since the padding is set to `1 rem` instead of the standard `.75rem`. You can check that this is the case by comparing the buttons to the "Preview as" buttons in the instructor edit session page.

In terms of the clarity of code, I think that adding the extra styling is fine since it's fairly straightforward to see what's going on. Maybe placing it within a class and then applying that to the button would make it clearer, something like:
```
@media (max-width: 576px) {
  // prevent overflow for small screen sizes
  .btn-small-padding {
    padding-left: .5rem;
    padding-right: .5rem;
  }
}
```
and then `<button id="btn-copy-question" class="btn btn-primary btn-small-padding"`

Let me know if you disagree. Also taking this chance to ping @domlimm for thoughts and review

Reply:
Okay I get it now. I thought the standard was `1rem`. Lemme quickly make the change.